docs/faqs.md

```diff
@@ -49,3 +53,74 @@ Workarounds before the fix is released:
 - do not mount `/dev`, provisioner will discover current mounted devices, but it cannot discovery the newly mounted (see [why](https://github.com/kubernetes-incubator/external-storage/issues/783#issuecomment-395013458))
+
+## Can I clean volume data by deleting PV object?
+
+No, there is no reliable mechanism in provisioner to detect the PV object of
+volume is deleted by the user or not created yet. This is because delete event
+will not be delivered when provisioner is not running and provisioner don't
+know whether the volume has been discovered before.
+
+So provisioner will always discover volume as a new PV if no existing PV is
+associated with it. The volume data will not be cleaned in this phrase, and old
+data in it may leak other applications.
+
+So you must not delete PV objects by yourself, always delete PVC objects and
+set `PersistentVolumeReclaimPolicy` to `Delete` to clean volume data of
+associated PVs.
+
+## PV with delete reclaimPolicy is released but not going to be reclaimed
+
+At first, please check provisioner is running on the node.  If provisioner is
+running, please check whether the volume exists on the node. If the volume is
+missing, provisioner can not clean the volume data. For safety, it will not
+clean the associated PV object. This is to prevent old volume from being used
+by other programs if it recovers later.
+
+It’s up to the system administrator to fix this:
+
+- If the volume has been decommissioned, you can delete the PV object manually
+  (e.g. `kubectl delete <pv-name>`)
+- If the volume is missing because of the invalid `/etc/fstab`, setup script or
+  hardware failures, please fix the volume. If the volume can be recovered,
+  provisioner will continue to clean the volume data and reclaim the PV object.
+  If the volume cannot be recovered, you can remove the volume (or disk) from
+  the node and delete the PV object manually.
+
+Of course, on a specific platform if you have a reliable mechanism to detect if
+a volume is permanently deleted or cannot recover, you can write an operator or
+sidecar to automate this.
+
+## Why my application uses an empty volume when node gets recreated in GCP
+
+Please check `spec.local.path` field of local PV object. If it is a non-unique
+path (e.g. without UUID in it) e.g. `/mnt/disks/ssd0`, newly created disks may be
+mounted at the same path.
+
+For example, in GKE when nodes get recreated, local SSDs will be recreated and
+mounted at the same paths (note: `--keep-disks` cannot be used to keep local
+SSDs because `autoDelete` cannot be set to false on local SSDs). Pods using old
+volumes will start with empty volumes because paths of PV objects will get
+mounted with newly created disks.
+
+If your application does not expect this behavior, you should use
+[`--local-ssd-volumes`](https://cloud.google.com/sdk/gcloud/reference/alpha/container/node-pools/create)
+and configure provisioner to discover volumes under `/mnt/disks/by-uuid/google-local-ssds-scsi-fs` or
```

Comment:
This example contains that config: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/helm/generated_examples/gce.yaml

Reply:
A link to this example is added.